gravity-forms/gw-all-fields-template.php

```diff
@@ -175,6 +175,7 @@ public function all_fields_extra_options( $value, $merge_tag, $modifiers, $field
 			 */
 			if ( $context == 'nested' ) {
 
+				$input_ids = array();
```

Comment:
Is the customer trying to get an input from the Nested Form? It seems like if this is the logic they're hitting we need to account for it in the foreach.

Reply:
Disregard, I see that they're not using input IDs for now. It would be a nice improvement at some point.